src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```
@@ -1073,6 +1079,23 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!--
+  ============================================================
+  Command Line Confirugation Validation
```

Typo which I missed in code review:
```suggestion
  Command Line Configuration Validation
```